code/modules/projectiles/updated_projectiles/guns/rifles.dm

```diff
@@ -37,53 +37,93 @@
 		return in_chamber ? (current_mag.current_rounds + 1) : current_mag.current_rounds
 
 //-------------------------------------------------------
-//M41A PULSE RIFLE
+//T-18 Carbine
 
 /obj/item/weapon/gun/rifle/standard_carabine
```

Comment:
```suggestion
/obj/item/weapon/gun/rifle/standard_carbine
```

Can you also fix the occurences of this?

Reply:
it has been done